osu.Game/Screens/Play/KeyCounter.cs

```diff
@@ -1,58 +1,56 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-#nullable disable
-
-using osu.Framework.Allocation;
+using osu.Framework.Bindables;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
-using osu.Framework.Graphics.Sprites;
-using osu.Framework.Graphics.Textures;
-using osu.Game.Graphics;
-using osu.Game.Graphics.Sprites;
-using osuTK;
-using osuTK.Graphics;
 
 namespace osu.Game.Screens.Play
 {
     public abstract partial class KeyCounter : Container
     {
-        private Sprite buttonSprite;
-        private Sprite glowSprite;
-        private Container textLayer;
-        private SpriteText countSpriteText;
+        public readonly InputTrigger Trigger;
 
-        public bool IsCounting { get; set; } = true;
-        private int countPresses;
+        protected Bindable<bool> IsCountingBindable = new BindableBool(true);
```

Comment:
Why is this a thing? It's protected, not accessed by any subclass, and it's only used in `IsCounting` which is... a plain bool property?